lib/server-impl.js

```diff
@@ -16,14 +14,14 @@ const AuthenticationRequired = require('./authentication-required');
 
 async function createApp(options) {
     // Database dependecies (statefull)
+    const logger = options.getLogger('server-impl.js');
     const stores = createStores(options);
     const eventBus = new EventEmitter();
 
     const config = Object.assign(
         {
             stores,
             eventBus,
-            logFactory,
```

Comment:
A user could potentially access this property through `stateService.config.logFactory`. I'd suggest moving this specific change to 4.x.

Reply:
Thanks, you are absolutely right!